lib/chatRepository.js

```diff
@@ -0,0 +1,30 @@
+const Redis = require('ioredis');
```

Comment:
`'use strict';` or GTFO

![](http://i.giphy.com/lfcaGJhOQst2M.gif)

no seriously because:
```shell
Users/fra/Workspace/node-telegram-sio-bot/lib/chatRepository.js:7
class ChatRepository {
^^^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:404:25)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/fra/Workspace/node-telegram-sio-bot/lib/sioBot.js:7:24)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
```